style.css

```diff
@@ -289,7 +286,7 @@ margin-top: 1px;
 .next-section_wrapper {
     display: flex;
     justify-content: center;
-    margin-bottom: 37px;
+    margin-bottom: 27px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Angleter_tesla/pullRequest?prid=5388479)